trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    super[0...9]
+  end
+end
```

Comment:
![image](https://github.com/Lawmsangi/school-library-oop/assets/93039359/93233374-c3b7-48a0-b9e2-6af3a0156a96)

- [ ] Great work so far 👍 but here is a thing that needs to be addressed 👀 You are supposed to return a maximum of 10 characters but your code is only returning 9 😨 Kindly make your code return 10 characters to follow the project requirements 😊

### Hint 🟢

Your code has three dots between 0 and 9 🤔 and there should be only two dots such as this `0...9` should be like this `0..9` 👍